homeassistant/components/fritzbox_netmonitor/config_flow.py

```diff
@@ -0,0 +1,75 @@
+"""Config flow for fritzbox_netmonitor."""
```

Comment:
A configuration flow requires tests with a 100% test coverage. Please add tests for this file to this PR.